bigbluebutton-html5/public/locales/en.json

```
@@ -634,10 +634,9 @@
     "app.actionsBar.actionsDropdown.takePresenterDesc": "Assign yourself as the new presenter",
     "app.actionsBar.actionsDropdown.selectRandUserLabel": "Select random user",
     "app.actionsBar.actionsDropdown.selectRandUserDesc": "Chooses a user from available viewers at random",
-    "app.actionsBar.interactions.interactions": "Interactions",
-    "app.actionsBar.interactions.raiseHand": "Raise your hand",
-    "app.actionsBar.interactions.lowHand": "Lower your hand",
-    "app.actionsBar.interactions.addReaction": "Add a reaction",
+    "app.actionsBar.reactions.reactionsButtonLabel": "Reactions",
```

```suggestion
    "app.actionsBar.reactions.reactionsButtonLabel": "Reactions bar",
```

After chatting with @ffdixon and @tylercopeland the decision reached was that calling it a Reactions **Bar** makes the most sense.